charts/jaeger/templates/collector-ing.yaml

```
@@ -7,7 +7,7 @@ kind: Ingress
 metadata:
   name: {{ template "jaeger.collector.name" . }}
   labels:
-    {{- include "jaeger.labels" . | nindent 4 }}
+    {{- toYaml .Values.collector.ingress.labels | nindent 4 }}
```

```suggestion
    {{- include "jaeger.labels" . | nindent 4 }}
    {{- if .Values.query.ingress.labels }}
    {{- toYaml .Values.collector.ingress.labels | nindent 4 }}
    {{- end }}
```

what about something like this that keeps the common labels, but allows folks to add additional labels to the ingress?

---

Totally agree, this is done.